mongo/xml.go

```diff
@@ -128,605 +128,605 @@ type Network struct {
-	TrackedBytesBelongingToLeafPagesInTheCache                 int `json:"tracked bytes belonging to leaf pages in the cache"`
-	TrackedDirtyBytesInTheCache                                int `json:"tracked dirty bytes in the cache"`
-	TrackedDirtyPagesInTheCache                                int `json:"tracked dirty pages in the cache"`
-	UnmodifiedPagesEvicted                                     int `json:"unmodified pages evicted"`
+	BytesBelongingToPageImagesInTheCache                       int `json:"bytes belonging to page images in the cache" bson:"bytes belonging to page images in the cache"`
+	BytesCurrentlyInTheCache                                   int `json:"bytes currently in the cache" bson:"bytes currently in the cache"`
+	BytesNotBelongingToPageImagesInTheCache                    int `json:"bytes not belonging to page images in the cache" bson:"bytes not belonging to page images in the cache"`
+	BytesReadIntoCache                                         int `json:"bytes read into cache" bson:"bytes read into cache"`
+	BytesWrittenFromCache                                      int `json:"bytes written from cache" bson:"bytes written from cache"`
+	CheckpointBlockedPageEviction                              int `json:"checkpoint blocked page eviction" bson:"checkpoint blocked page eviction"`
+	EvictionCallsToGetAPage                                    int `json:"eviction calls to get a page" bson:"eviction calls to get a page"`
+	EvictionCallsToGetAPageFoundQueueEmpty                     int `json:"eviction calls to get a page found queue empty" bson:"eviction calls to get a page found queue empty"`
+	EvictionCallsToGetAPageFoundQueueEmptyAfterLocking         int `json:"eviction calls to get a page found queue empty after locking" bson:"eviction calls to get a page found queue empty after locking"`
+	EvictionCurrentlyOperatingInAggressiveMode                 int `json:"eviction currently operating in aggressive mode" bson:"eviction currently operating in aggressive mode"`
+	EvictionEmptyScore                                         int `json:"eviction empty score" bson:"eviction empty score"`
+	EvictionServerCandidateQueueEmptyWhenToppingUp             int `json:"eviction server candidate queue empty when topping up" bson:"eviction server candidate queue empty when topping up"`
+	EvictionServerCandidateQueueNotEmptyWhenToppingUp          int `json:"eviction server candidate queue not empty when topping up" bson:"eviction server candidate queue not empty when topping up"`
+	EvictionServerEvictingPages                                int `json:"eviction server evicting pages" bson:"eviction server evicting pages"`
+	EvictionServerSleptBecauseWeDidNotMakeProgressWithEviction int `json:"eviction server slept\, because we did not make progress with eviction" bson:"eviction server slept\, because we did not make progress with eviction"`
```

Comment:
To reviewers:
The comma needs to be escaped,. Otherwise, it will be treat as separator after which some flag is expected to be provided (e.g. omitempty)